src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -184,12 +246,10 @@ const resolveLandingPageLinks = (
     {
       title: HEALTH_TOOL_HEADINGS.MEDICAL_RECORDS,
       icon: 'note_add',
-      ...(!featureToggles[
-        FEATURE_FLAG_NAMES.mhvLandingPageShowShareMyHealthDataLink
-      ] && {
+      ...{
```

Comment:
I think we can remove this object+spread expression, since we're now creating an object with a single property and immediately "unwrapping" it. 

I think that mean removing lines 249 and 252 and fixing indentation for 251-252…